jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -57,12 +57,18 @@
             status-url: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/'
             status-add-test-results: '{status-add-test-results}'
             # This should roughly line up with kubernetes-maintainers.
+            # To update:
+            #   curl -s "https://api.github.com/teams/1674435/members?per_page=100&access_token=TOKEN" \
```

Comment:
It looks like the list is almost 100 users long. Once that happens this command will drop people.